src/Analysis.cpp

```diff
@@ -19,6 +20,13 @@
 // note for run all tests is that if you add more tests, you'll have to modify
 // run_all.sh to include that test number
 
+// if a known function is being re-defined, issue a warning and do not check its
+// annotations. treat it as is we want to remove UnsafeFunctions and just have
```

Comment:
This comment won't make sense once this PR is merged, I think, because you've removed all other mentions of `UnsafeFunctions` from the code. Can you rewrite it so that it makes sense without the reader needing to be familiar with the previous version of the code?

Reply:
Changing comment to "if a known function is being re-defined, issue a warning and remove it from safe/realloc/memory functions (wherever it's in). it's annotations should be checked (once annotations are fully implemented)"